drawBot/macOSVersion.py

```diff
@@ -0,0 +1,6 @@
+from distutils.version import StrictVersion
+import platform
+
+# It is safe to compare osVersion to strings, as StrictVersion casts strings
+# to StrictVersion instances upon compare.
+macOSVersion = StrictVersion(platform.mac_ver()[0])
```

Comment:
Check if the platform is not None and use a fallback instead

See `AttributeError: 'StrictVersion' object has no attribute 'version'` on read the docs

Reply:
Yes, I think I found a solution.

Reply:
Does RTD build PR branches? If not, we may have to merge, and continue on master if it still doesn't work.

Reply:
Ah right, it builds "stable" and "latest". I'm fairly confident this PR will fix the RTD build, though, so I think we should merge (once Travis is done :) ).